orcid-persistence/src/main/resources/db/updates/reindex_locked_deprecated_and_deactivated_records.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    =============================================================================
+
+    ORCID (R) Open Source
+    http://orcid.org
+
+    Copyright (c) 2012-2014 ORCID, Inc.
+    Licensed under an MIT-Style License (MIT)
+    http://orcid.org/open-source-license
+
+    This copyright and license information (including a link to the full license)
+    shall be included in its entirety in all copies or substantial portion of
+    the software.
+
+    =============================================================================
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+	<changeSet id="REINDEX-LOCKED-DEPRECATED-AND-DEACTIVATED-RECORDS" author="Angel Montenegro" dbms="postgresql">
+		<sql>UPDATE profile SET indexing_status='REINDEX' WHERE orcid IN (SELECT orcid FROM profile WHERE record_locked = true OR primary_record IS NOT NULL OR profile_deactivation_date IS NOT NULL);</sql>
```

Comment:
I'm a bit confused what happens if this is deployed on more than one node?

Reply:
Hey @TomDemeranville, every liquibase scripts runs in one node and lock the database, when it finish, it mark the script as ran and release the lock, so, the next node will wait until the first one finishes, and then it will notice that the script already ran and will just ignore it.

Reply:
Awesome.